packages/matter.js/src/cluster/ColorControlCluster.ts

```diff
@@ -0,0 +1,734 @@
+        primary1X: OptionalAttribute(0x0011, TlvUInt16.bound({ max: 0xfeff })),
+        primary1Y: OptionalAttribute(0x0012, TlvUInt16.bound({ max: 0xfeff })),
+        primary1Intensity: OptionalAttribute(0x0013, TlvNullable(TlvUInt8)),
+        primary2X: OptionalAttribute(0x0015, TlvUInt16.bound({ max: 0xfeff })),
+        primary2Y: OptionalAttribute(0x0016, TlvUInt16.bound({ max: 0xfeff })),
+        primary2Intensity: OptionalAttribute(0x0017, TlvNullable(TlvUInt8)),
+        primary3X: OptionalAttribute(0x0019, TlvUInt16.bound({ max: 0xfeff })),
+        primary3Y: OptionalAttribute(0x001A, TlvUInt16.bound({ max: 0xfeff })),
+        primary3Intensity: OptionalAttribute(0x001B, TlvNullable(TlvUInt8)),
+
+        /** 
+         * Additional Defined Primaries Information Attribute Set
+         * @see {@link MatterApplicationClusterSpecificationV1_0} § 3.2.9 
+         */
+        primary4X: OptionalAttribute(0x0020, TlvUInt16.bound({ max: 0xfeff })),
+        primary4Y: OptionalAttribute(0x0021, TlvUInt16.bound({ max: 0xfeff })),
+        primary4Intensity: OptionalAttribute(0x0022, TlvNullable(TlvUInt8)),
+        primary5X: OptionalAttribute(0x0024, TlvUInt16.bound({ max: 0xfeff })),
+        primary5Y: OptionalAttribute(0x0025, TlvUInt16.bound({ max: 0xfeff })),
+        primary5Intensity: OptionalAttribute(0x0026, TlvNullable(TlvUInt8)),
+        primary6X: OptionalAttribute(0x0028, TlvUInt16.bound({ max: 0xfeff })),
+        primary6Y: OptionalAttribute(0x0029, TlvUInt16.bound({ max: 0xfeff })),
+        primary6Intensity: OptionalAttribute(0x002A, TlvNullable(TlvUInt8)),
+
+        /** 
+         * Defined Color Points Settings Attribute Set
+         * @see {@link MatterApplicationClusterSpecificationV1_0} § 3.2.10 
+         */
+        whitePointX: OptionalAttribute(0x0030, TlvUInt16.bound({ max: 0xfeff })),
+        whitePointY: OptionalAttribute(0x0031, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointRX: OptionalAttribute(0x0032, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointRY: OptionalAttribute(0x0033, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointRIntensity: OptionalAttribute(0x0034, TlvNullable(TlvUInt8)),
+        colorPointGX: OptionalAttribute(0x0036, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointGY: OptionalAttribute(0x0037, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointGIntensity: OptionalAttribute(0x0038, TlvNullable(TlvUInt8)),
+        colorPointBX: OptionalAttribute(0x003A, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointBY: OptionalAttribute(0x003B, TlvUInt16.bound({ max: 0xfeff })),
+        colorPointBIntensity: OptionalAttribute(0x003C, TlvNullable(TlvUInt8)),
+    },
+});
+
+const hueSaturationAttributes = {
+    currentHue: Attribute(0x0, TlvUInt8.bound({ max: 0xfe }), { default: 0x0, persistent: true }), // This and following are Mandatory if HueSat flag set in features
```

Comment:
add "scene: true" please,also for the others where specs tell "S" (mainly the current* ones and yes specs do not tell for hue but that makes no sense, so we also add it there.)